data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/cnec/CsaProfileCnecCreator.java

```diff
@@ -109,13 +108,23 @@ private void addCnec(PropertyBag assessedElementPropertyBag) {
             }
         } else if (CsaProfileConstants.LimitType.VOLTAGE.equals(limitType)) {
             cnecAdder = crac.newVoltageCnec()
-                .withMonitored(true)
+                .withMonitored(false)
```

Comment:
this should be true if you re-merge the last commits from master:
```suggestion
                .withMonitored(true)
```